server/issue.go

```diff
@@ -44,6 +46,18 @@ func (s *Server) registerIssueRoutes(g *echo.Group) {
 	g.GET("/issue", func(c echo.Context) error {
 		ctx := c.Request().Context()
 		issueFind := &api.IssueFind{}
+
+		if pageToken := c.QueryParams().Get("token"); pageToken != "" {
```

Comment:
Let's have unmarshalPageToken() to handle this when c.QueryParams().Get("token") == "".

Reply:
Done.